src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,52 @@
+/* eslint-disable max-len */
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie,
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <div className="card" data-cy="Movie" key={movie.imdbId}>
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={movie.imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8" data-cy="MovieTitle">
+            {movie.title}
+          </p>
+        </div>
+      </div>
+
+      <div className="content">
+        <p data-cy="MovieDescription">
+          {movie.description}
+        </p>
+
+        <a
+          href="https://www.imdb.com/title/tt1375666"
+          data-cy="MovieLink"
+        >
+          {movie.imdbUrl}
+        </a>
```

Comment:
![image](https://user-images.githubusercontent.com/59209337/230561944-1798ed72-3abe-41c5-9f2a-64615938209a.png)

Reply:
url should be diffenet for each movie, but the title should be the same